product_docs/docs/pem/8/pem_online_help/04_toc_pem_features/12_pem_manage_probes/03_pem_probe_config/01_pem_probes.mdx

```diff
@@ -68,7 +74,10 @@ Unless otherwise noted, Postgres Enterprise Manager enables the probes listed in
 
 ## BDR Probes
 
-To monitor the BDR Group via [BDR dashboards](../../01_dashboards/03_bdr_dashboard/#bdr_dashboard) the following probes must be enabled. All these probes are configured at server level.
+To monitor the BDR Group via [BDR dashboards](../../01_dashboards/03_bdr_dashboard/#bdr_dashboard) the following probes must be enabled. All these probes are configured at extension level.
+
+! Note
+  Prior to version 8.4, all these probes are available at server level.
```

Comment:
does this mean that these probes are only available at the extension level now or both server and extension?

Reply:
Prior to version 8.4, all these probes are available at the server level only. From 8.4 onwards they are available at the only extension level.